tests/ignite/metrics/test_precision_recall_curve.py

```diff
@@ -28,16 +35,17 @@ def test_no_sklearn(mock_no_sklearn):
         pr_curve.compute()
 
 
-def test_precision_recall_curve():
+def test_precision_recall_curve(available_device):
     size = 100
     np_y_pred = np.random.rand(size, 1)
     np_y = np.zeros((size,))
     np_y[size // 2 :] = 1
     sk_precision, sk_recall, sk_thresholds = precision_recall_curve(np_y, np_y_pred)
 
-    precision_recall_curve_metric = PrecisionRecallCurve()
-    y_pred = torch.from_numpy(np_y_pred)
-    y = torch.from_numpy(np_y)
+    precision_recall_curve_metric = PrecisionRecallCurve(device=available_device)
+    assert precision_recall_curve_metric._device == torch.device(available_device)
+    y_pred = to_tensor(np_y_pred, available_device)
```

Comment:
I would prefer to create the data directly with torch tensors instead of numpy arrays. For expected values computations you can do as usual `.cpu().numpy()`

Reply:
Did I understand correctly what needs to be done?